### ISSUE TITLE
No compatible version found: linkifyjs@'*'

### ISSUE THREAD

**Issue by arasmussen** [SoapBox/linkifyjs#75]
```bash
$ npm install --save linkifyjs
npm ERR! Darwin 14.4.0
npm ERR! argv "/usr/local/bin/iojs" "/usr/local/bin/npm" "install" "linkifyjs" "--save"
npm ERR! node v2.3.4
npm ERR! npm  v2.12.1
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: linkifyjs@'*'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["2.0.0-alpha.3","2.0.0-alpha.4","2.0.0-beta.1","2.0.0-beta.2","2.0.0-beta.3","2.0.0-beta.4"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'my-project'
npm ERR! notarget
```

Any ideas how I can get this working with iojs?

**Comment by nfrasser** [comment]
Hi @arasmussen, I was able to reproduce this as well with your setup. Looks like this is a bug with NPM (see here: https://github.com/npm/npm/issues/8951)

I was able to get it to work with this command:

```
npm install --save linkifyjs@2.0.0-beta.4
```

This should be fixed once 2.0 is officially released.

**Comment by nfrasser** [comment]
Fixed as of NPM v2.14.2

**Comment by arasmussen** [comment]
Thanks!
